staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -251,7 +251,7 @@ func DefaultOpenAPIConfig(getDefinitions openapicommon.GetOpenAPIDefinitions, sc
 // WebServices set.
 func DefaultSwaggerConfig() *swagger.Config {
 	return &swagger.Config{
-		ApiPath:         "/swaggerapi/",
+		ApiPath:         "/swaggerapi",
```

Comment:
is this fixing a double-slash issue, or changing consumption of subpaths, or something else?